space-framework/docs/space-framework.md

```diff
@@ -0,0 +1,92 @@
+# Space Framework
+
+## terminoloogy 
+**Physical Space**, **pSpace** A thing that behaves like a Kubernetes kube-apiserver (including the persistent storage behind it) and the subset of controllers in the kube-controller-manager that are concerned with API machinery generalities (not management of containerized workloads). A kcp logical cluster is an example. A regular Kubernetes cluster is another example.
+
+**Space Object**, **Space** - A kind of Kubernetes API object that _represents_ a pSpace.  Its "kind" is `Space`, we sometimes include the "object" part when writing specifically about the API objects to clearly distinguish them from what they represent.
+
+**Space Provider** - A Thing that manages the life cycle of multiple pSpaces. This includes at minimal create/delete/List operations on pSpaces. KIND is an example for Space Provider (that manages KIND clusters), KCP is another example - KCP manages KCP's workspaces. 
+
```

Comment:
need to rephrase again and not use "Thing" ... 
maybe: An entity that manages the life cycle of multiple pSpaces. This entails, at the very least, performing create/delete/list operations on pSpaces. examples: KIND is a space provider that manages KIND cluster, and KCP manages KCP's workspaces.